include/interface/cmdTlmServer.h

```diff
@@ -0,0 +1,101 @@
+#ifndef CMDTLMSERVER_H
+#define CMDTLMSERVER_H
```

Comment:
Since I have another class: `TcpServer` in this repository, the use of `Server` name might be confusing... (too general). Please let me use this name. Thanks!